.circleci/config.yml

```diff
@@ -187,6 +128,7 @@ jobs:
       - restore_dependency_cache
       - browser-tools/install-chrome
       - browser-tools/install-chromedriver
+      - run: npm install -g npm@7
```

Comment:
Why pin this at v7? I imagine this will be easy to overlook when we upgrade Node.js.

Reply:
A few lines above that we we're using a node14 image due to WDIO. Node 14 comes with npm 6.14, which doesn't yet support NPM workspaces.